packages/plugin-indent/src/Indent.ts

```diff
@@ -153,7 +153,7 @@ export class Indent<T extends JWPluginConfig = JWPluginConfig> extends JWPlugin<
      * @param params
      */
     _isSegmentBreak(node: VNode): boolean {
-        return node instanceof ContainerNode || node instanceof LineBreakNode;
+        return (node instanceof ContainerNode && node.tangible) || node instanceof LineBreakNode;
```

Comment:
Why is it required as node.tangible should be filtered by default, no ?

Reply:
oups ^^ ...a commit error when I tested